### DISCUSSION TITLE
Velero backup and restore on different EKS versions

### DISCUSSION THREAD

**Discussion by rizwank23** [vmware-tanzu/velero#8566] [General discussions]
Hello !!!

We have an EKS cluster with the version 1.24 which has node group with 1.20. Now we want to try Velero here to take the backup and restore on EKS cluster with the version 1.30

Will this scenario work as we have different EKS versions?

Thanks
Rizwan

**Comment by blackpiglet** [comment]
Velero should work in this scenario, which is not covered in the Velero E2E test cases.
Please have a test before doing the backup and restore in the production environment.

This is the compatibility matrix of Velero and the k8s versions
https://github.com/vmware-tanzu/velero?tab=readme-ov-file#velero-compatibility-matrix.
The v1.24 and the v1.30 k8s are covered.
